mmdeploy/codebase/mmedit/deploy/inpainting.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
+from typing import Callable, Optional, Sequence, Tuple, Union
+
+import mmcv
+import numpy as np
+import torch
+import torch.nn as nn
+from mmcv.parallel import collate, scatter
+from mmcv.utils import Config
+from torch.utils.data import Dataset
+
+from mmdeploy.codebase.base import BaseTask
+from mmdeploy.codebase.mmedit.deploy.mmediting import MMEDIT_TASK
+from mmdeploy.utils import Task, get_ir_config, load_config
+
+
+def process_model_config(model_cfg: Config,
+                         imgs: Union[Sequence[str], Sequence[np.ndarray]],
+                         input_shape: Optional[Sequence[int]] = None):
+    """Process the model config.
+
+    Args:
+        model_cfg (Config): The model config.
+        imgs (Sequence[str] | Sequence[np.ndarray]): Input image(s), accepted
+            data type are List[str], List[np.ndarray].
+        input_shape (Sequence[int], optional): A list of two integer
+            in (width, height) format specifying input shape. Default: None.
+
+    Returns:
+        Config: the model config after processing.
+    """
+    config = load_config(model_cfg)[0].copy()
+    load_from_file = isinstance(imgs[0], str)
+    if not load_from_file:
+        # Remove 'LoadImageFromFile'
+        config.test_pipeline.pop(0)
+
+    is_static_cfg = input_shape is not None
+    if is_static_cfg:
+        assert 'Resize' in [_.type for _ in config.test_pipeline]
+        for pipeline in config.test_pipeline[::-1]:
+            if pipeline.type == 'Resize':
+                pipeline.scale = input_shape
```

Comment:
This line results in throwing exception when converting the pytorch model to its tensorrt model in one command line
```
 python tools/deploy.py configs/mmedit/inpainting/inpainting_tensorrt_static-256x256.py /data1/checkpoint/mmedit/deepfillv2/deepfillv2_256x256_8x2_places.py /data1/checkpoint/mmedit/deepfillv2/deepfillv2_256x256_8x2_places_20200619-10d15793.pth --device cuda demo/resources/det.jpg --work-dir /data1/mmdeploy_models/mmedit/trt/deepfillv2 
```

That's because `pipeline.scale` of `Resize` in mmediting is requested to be `tuple` instead of `list`